aws/resource_aws_lambda_alias_test.go

```diff
@@ -219,8 +223,7 @@ func testAccCheckAwsLambdaAliasRoutingConfigDoesNotExist(mapping *lambda.AliasCo
 		return nil
 	}
 }
-
-func testAccAwsLambdaAliasConfig(roleName, policyName, attachmentName, funcName, aliasName string) string {
+func testAccAwsLambdaAliasBaseConfigWithConfig(roleName, policyName, attachmentName, config string) string {
```

Comment:
We recently introduced a tiny helper function to concatenate test configuration strings that we would like to standardize on going forward (we owe the repository Contributing Guide an update and an issue for refactoring for consistency) 👍 

```
return composeConfig(
	config1(),
	fmt.Sprintf(``),
)
```

Reply:
I've updated the code to use the helper function

Reply:
Thanks so much!